### ISSUE TITLE
animate() docstring refers to non-existing with_viewer option (under scale_factor)

### ISSUE THREAD

**Issue by anntzer** [napari/napari-animation#146]
Perhaps you actually meant canvas_only = True?

**Comment by alisterburt** [comment]
Good catch! Would accept a PR if you're interested in fixing, otherwise will get to it later this week 🙂

**Comment by anntzer** [comment]
I'll let you have a go :)

**Comment by psobolewskiPhD** [comment]
If anyone is looking, the docstring to fix is here:
https://github.com/napari/napari-animation/blob/a9ca7a7a96e5b281bf80815aaded8a6ce0b72872/napari_animation/animation.py#L157
